tensorbay/opendataset/CACD/loader.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+# pylint: disable=missing-module-docstring
+
+import os
+from typing import Any, Dict, Tuple
+
+import numpy as np
+
+from ...dataset import Data, Dataset
+from ...exception import ModuleImportError
+from ...label import CategoryInfo, Classification, ClassificationSubcatalog
+from ...utility import NameList
+
+DATASET_NAME = "CACD"
+_ATTRIBUTES = {
+    "attributes": [
+        {"type": "number", "minimum": 14, "maximum": 62, "name": "age"},
+        {"type": "number", "minimum": 1951, "maximum": 1990, "name": "birth"},
+        {"type": "number", "minimum": 1, "maximum": 50, "name": "range"},
+        {"type": "number", "minimum": 2004, "maximum": 2013, "name": "year"},
+        {"type": "string", "name": "name"},
+        {
+            "type": "boolean",
+            "name": "lfw",
+            "description": "Whether the celebrity is in LFW dataset.",
+        },
+    ]
+}
+
+
+def CACD(path: str) -> Dataset:
+    """Dataloader of `Cross-Age Celebrity Dataset (CACD)`_ dataset.
+
+    .. _Cross-Age Celebrity Dataset (CACD): https://bcsiriuschen.github.io/CARC/
+
+    The file structure should be like::
+
+        <path>
+            CACD2000/
+                14_Aaron_Johnson_0001.jpg
+                ...
+            celebrity2000.mat
+
+    Arguments:
+        path: The root directory of the dataset.
+
+    Returns:
+        Loaded :class:`~tensorbay.dataset.dataset.Dataset` instance.
+    """
```

Comment:
normally the input path should be transferred to abspath. Check other loaders as reference